ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -421,3 +428,35 @@ def pytest_runtest_makereport(item, call):
             collect_performance_stats(test_case_name)
         except Exception as ex:
             log.error(f"Failed to collect performance stats. Error: {ex}")
+
+
+def set_report_portal_tags(config):
+    rp_tags = list()
+    rp_tags.append(ocsci_config.ENV_DATA.get('platform'))
+    rp_tags.append(ocsci_config.ENV_DATA.get('deployment_type'))
+    if ocsci_config.REPORTING.get('us_ds') == 'us':
+        rp_tags.append('upstream')
+    else:
+        rp_tags.append('downstream')
+    worker_instance_type = ocsci_config.ENV_DATA.get('worker_instance_type')
+    rp_tags.append(f"worker_instance_type:{worker_instance_type}")
+    rp_tags.append(f"ocp_version:{get_ocp_version()}")
+    rp_tags.append(
+        f"ocs_version:{ocsci_config.ENV_DATA.get('ocs_version')}"
+    )
+    if ocsci_config.DEPLOYMENT.get('ocs_registry_image'):
+        ocs_registry_image = ocsci_config.DEPLOYMENT.get('ocs_registry_image')
+        rp_tags.append(f"ocs_registry_image:{ocs_registry_image}")
+        rp_tags.append(f"ocs_registry_tag:{ocs_registry_image.split(':')[1]}")
+    if ocsci_config.DEPLOYMENT.get('ui_deployment'):
+        rp_tags.append('ui_deployment')
+    if ocsci_config.DEPLOYMENT.get('live_deployment'):
+        rp_tags.append('live_deployment')
+    if ocsci_config.DEPLOYMENT.get('stage'):
```

Comment:
if you can also add one more tag for fips please will be great.  Value you can take from:  https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/fips.yaml

Reply:
Done!